File: src/main/java/mods/eln/sim/mna/misc/MnaConst.java
package mods.eln.sim.mna.misc;

public class MnaConst {
    // Note: Avoid using ultraImpedance in grid tech, as it causes instability above 5 kV. Use highImpedance instead.
    public static final double ultraImpedance = 1e16;
    public static final double highImpedance = 1e9;
    public static final double pullDown = 1e9;
    public static final double noImpedance = 1e-9;
}

File: src/main/java/mods/eln/sim/mna/misc/MnaConst.kt
package mods.eln.sim.mna.misc

import mods.eln.misc.MaterialProperties
import mods.eln.misc.MaterialType

object MnaConst {
    val ultraImpedance = 1e16
    val highImpedance = 1e9
    val pullDown = 1e9
    val noImpedance = MaterialProperties.getElectricalResistivity(MaterialType.COPPER) * (25.0 / 1000000.0 / 1.0) * 10_000_000_000
}
